src/Controller/ValetToken/ValetTokenController.ts

```diff
@@ -0,0 +1,39 @@
+import { inject } from 'inversify'
+import { 
+  BaseHttpController, 
+  controller, 
+  httpPost, 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
hmm can you try changing the `yarn test` command to have `--no-cache` param and see if this still pops up as unused? https://jestjs.io/docs/cli#--cache

Reply:
This is done because the only place `results` are used is in the return type of the `create` method. The linter is ignoring the type annotations. 

The same thing happens in `auth` (e.g. in `AuthController`) which is where I got this from. I see no better way to do it.